### PR TITLE
[CR] Reworking Brewing

### PR DESCRIPTION
**Pull request by Korhaka**
#### Summary
SUMMARY: Balance "Updating brewing recipes and fermentation times"

#### Purpose of change
Brewing recipes often take more time crafting manually and less time fermenting compared with reality. 

#### Describe the solution
Drinks like whiskey and beer are now made with malted grains, a process which takes several days while the grains germinate. Similarly with dandelion wine that requires boiling (30 minutes) then leaving to soak for 2 hours before yeast can be added. 

Higher batch time multiplier as brewing recipes are generally throwing any quantity of liquid into a larger container. Recipes that involve simply mixing ingredients together rather than cooking (mead, fruit wine, dandelion wine after soaking the dandelions) have much shorter recipe duration as they are just a case of mixing ingredients together rather than any cooking. Fruit wine now only craftable from fruit juice with yeast being added directly to it.

Fermenting times are generally increased to 7 days, mead to 14 days.

#### Describe alternatives you've considered
As a large number of things can be fermented, often creating something that may be a mix between two different drinks, eg. honey and apple juice would create something between mead and cider. One option could have been making 2 difference brewing recipes, one for mixing cold ingredients together and another for cooking ingredients together. Then fermenting them and a resulting alcoholic drink. The result then could be distilled again into a fairly generic drink, or ethanol.

Downsides of this would be that even if conditional making persisted through brewing, values such as fun would be the same for all drinks, from a sugar wash (pretty disgusting) to a fine mead (delicious). 

#### Testing
Test that recipes are showing up and drinks with altered ingredients (malted grains, soaked dandelions) are craftable with the new ingredients. 

#### Additional context
Brewing Notes:
Malted Grain, mix with water and age for few days

Distilled: (Cooked)
Moonshine - corn
Vodka - wheat barley buckwheat
Whiskey - malted grain
Rum - molasses, sugar beets

Beer - malted grain (cook)
Fruit wine - fruit juice (quick mix)
Mycus wine (juice and mix? how is this actually made)
Mead (slightly slower mix as honey disolves)
Dandelion wine (quick cook, longer aged seeping)
Pine wine (cook)

Additionally - As far as real life testing I have brewed cider (apple juice + yeast) and mead (honey + water + yeast) to 6-10.5% ABV. This is what gave me the idea to reduce recipe times for these two recipes (Fruit wine ingame for cider). I know meads take longer to ferment generally due to the antimicrobial properties in honey.

### CODE REVIEW

File: data/json/recipes/food/brewing.json [hunk 1]
```diff
@@ -151,113 +144,109 @@
...
     "result": "brew_hb_beer",
-    "result_mult": 15,
+    "result_mult": 10,
     "category": "CC_FOOD",
     "subcategory": "CSC_FOOD_BREW",
     "skill_used": "cooking",
     "difficulty": 4,
-    "time": "80 m",
-    "batch_time_factors": [ 50, 4 ],
+    "time": "30 m",
```
Comment (Reclusive-reptile) [inline data/json/recipes/food/brewing.json:178]: 60 minutes would be better here. 30 minutes to mash the grains followed by a 30 minute boil to isomerize the hop oils.

File: data/json/recipes/food/brewing.json [hunk 2]
```diff
@@ -267,24 +256,56 @@
...
+    "result_mult": 1,
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_BREW",
+    "skill_used": "cooking",
+    "difficulty": 3,
+    "time": "30 s",
+    "batch_time_factors": [ 95, 4 ],
+    "book_learn": [ [ "brewing_cookbook", 4 ] ],
+    "qualities": [ { "id": "CONTAIN", "level": 1 } ],
+    "components": [ [ [ "water", 1 ], [ "water_clean", 1 ] ], [ [ "corn", 1 ], [ "barley", 1 ], [ "buckwheat", 1 ], [ "oats", 1 ] ] ]
```
Comment (Reclusive-reptile) [inline data/json/recipes/food/brewing.json:291]: Please add "wheat" to the possible grains.

File: data/json/recipes/food/brewing.json (hunk 1 above)
Comment (Korhaka) [reply data/json/recipes/food/brewing.json:178]: Seems reasonable, what about the batch time? Boiling part should be a reasonable batch time although mashing would be labour intensive and more involved for a larger quantity.

File: data/json/recipes/food/brewing.json (hunk 2 above)
Comment (Korhaka) [reply data/json/recipes/food/brewing.json]: Added

**Comment by pjf** [comment]
@Korhaka : Thank you so much for these!

I'm playing with these changes on a local build, which means I've resolved all the conflicts (mostly book changes) with `CleverRaven:master` in [`pjf:44806_unconflicted_brewing`](https://github.com/pjf/Cataclysm-DDA/tree/44806_unconflicted_brewing). I could probably rebase it to 0.F if desired, since we're in string-freeze right now.

However I'm not sure of the etiquette here. I'm happy to open a new PR (all commits are attributed to @Korhaka), for `Korhaka:master` to be replaced with my conflict-resolved version, or for my conflict-resolved branch to just be hanging out.

In any case, I'm really happy to have found these, as they make brewing much more immersive. :beer:

**Comment by Korhaka** [comment]
@pjf Yeah that would be fine with me, anything that gets the changes into the game (and thus, more post-apocalyptic brewing) is good with me!

**Comment by pjf** [comment]
@Korhaka : Done in #49512! Thank you!

**Pull request closed without merging**